postgres/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3'
```

Comment:
I really like the extensive documentation on provisioning the database.

However, do you think consumers of this plugin would find this level of setup instructions useful? I feel it could fall outside the scope of this project.